dask_ml/model_selection/_incremental.py

```diff
@@ -166,6 +166,14 @@ async def _fit(
     models: Dict[int, Tuple[Model, Meta]] = {}
     scores: Dict[int, Meta] = {}
 
+    if "partial_fit" not in dir(model):
```

Comment:
```suggestion
    if not hasattr(model, "partial_fit"):
```

This seems more idiomatic to me.  It's a very slight comment though